pkg/kubectl/delete.go

```diff
@@ -203,8 +203,8 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string, timeout
 			return err
 		}
 	}
-	falseVar := false
-	deleteOptions := &metav1.DeleteOptions{OrphanDependents: &falseVar}
+	policy := metav1.DeletePropagationBackground
```

Comment:
Could you add a comment saying we choose "background" here because the rc has already been scaled down? And for the other places in this file.